src/main/java/org/apache/maven/plugin/doap/DoapUtil.java

```diff
@@ -472,16 +471,13 @@ public static void fetchURL(Settings settings, URL url) throws IOException {
         }
 
         if ("file".equals(url.getProtocol())) {
-            InputStream in = null;
-            try {
-                in = url.openStream();
-                in.close();
-                in = null;
-            } finally {
-                IOUtil.close(in);
+            // [ERROR] src/main/java/org/apache/maven/plugin/doap/DoapUtil.java:[474,53] (blocks) EmptyBlock: Empty try
+            // block.
+            try (InputStream in = url.openStream()) {
+                return;
+            } catch (IOException ex) {
+                return;
```

Comment:
what does code do ❓

Reply:
I think it just checks that the file exists.

Reply:
but exit from method always ....  it will be the same:

```java
if ("file".equals(url.getProtocol())) {
   return;
}
```

Reply:
You're right, fixed